vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -1,17 +1,142 @@
 <template>
-  
+  <div class="listing-form"> 
+    <div class="listing-title"> 
+      <v-text-field class="listing-title-entry"
+      label = "Listing Title"
+      filled>
+      </v-text-field>
+    </div>
+    <div class="listing-address">
+      <v-text-field class="listing-address-entry"
+        solo
+        label="Street Address"
+        prepend-inner-icon="mdi-map-marker"
+        filled>
+      </v-text-field>
+      <v-text-field class="listing-city-entry"
+        solo
+        label="City"
+        filled>
+      </v-text-field>
+      <v-text-field class="listing-state-entry"
+        solo
+        label="State/Province"
+        filled>
+      </v-text-field>
+    </div>
+    <!--Insert Image Components Here/Drag and Drop-->
+
+    <div class="start-date">
+      <v-menu
+          ref="menu"
+          v-model="menu"
+          :close-on-content-click="false"
+          :return-value.sync="date"
+          transition="scale-transition"
+          offset-y
+          min-width="290px">
+          <template v-slot:activator="{ on, attrs }">
+            <v-text-field
+              v-model="date"
+              label="Available Start Date"
+              prepend-icon="event"
+              readonly
+              v-bind="attrs"
+              v-on="on"
+            ></v-text-field>
+          </template>
+          <v-date-picker v-model="date" no-title scrollable>
+            <v-spacer></v-spacer>
+            <v-btn text color="primary" @click="menu = false">Cancel</v-btn>
+            <v-btn text color="primary" @click="$refs.menu.save(date)">OK</v-btn>
+          </v-date-picker>
+        </v-menu>
+    </div>
+
+    <div class="end-date">
+      <v-menu
+          ref="menu"
+          v-model="menu"
+          :close-on-content-click="false"
+          :return-value.sync="date"
+          transition="scale-transition"
+          offset-y
+          min-width="290px"
+        >
+          <template v-slot:activator="{ on, attrs }">
+            <v-text-field
+              v-model="date"
+              label="Available End Date"
+              prepend-icon="event"
+              readonly
+              v-bind="attrs"
+              v-on="on"
+            ></v-text-field>
+          </template>
+          <v-date-picker v-model="date" no-title scrollable>
+            <v-spacer></v-spacer>
+            <v-btn text color="primary" @click="menu = false">Cancel</v-btn>
+            <v-btn text color="primary" @click="$refs.menu.save(date)">OK</v-btn>
+          </v-date-picker>
+        </v-menu>
+    </div>
+    
+    <div class="description-entry">
+          <v-textarea
```

Comment:
Can we changes spaces from 4 -> 2? I also like 4 spaces but the Google HTML/CSS styleguide asks for 2
https://google.github.io/styleguide/htmlcssguide.html

Reply:
Will do!